snmp/datadog_checks/snmp/data/profiles/apc_ups.yaml

```diff
@@ -54,6 +54,62 @@ metrics:
     symbol:
       OID: 1.3.6.1.4.1.318.1.1.1.7.2.3
       name: upsAdvTestDiagnosticsResults
+  - MIB: PowerNet-MIB
+    symbol:
+      OID: 1.3.6.1.4.1.318.1.1.1.11.1
```

Comment:
Value should be `1.3.6.1.4.1.318.1.1.1.11.1.1.0`.

Reason: https://github.com/DataDog/integrations-core/pull/7105